### PR TITLE
FEAT: Remove python_src directory from repository

### PR DESCRIPTION
**Pull request by rymarczy**
Remove `python_src` directory and move all files/folders inside of it into root project directory.

Asana Task: https://app.asana.com/0/1205827492903547/1205970072942237

### CODE REVIEW

File: .github/workflows/ci_python.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/python_deps
 
-      - run: poetry run pylint src tests
+      - run: poetry run pylint src tests --rcfile pyproject.toml
```
Comment (rymarczy) [inline .github/workflows/ci_python.yaml:63]: Can't get pylint to pick up the config in `pyproject.toml` without this...

Open to suggestions on an alternate fix.

File: .gitignore [hunk 2]
```diff
@@ -20,7 +20,6 @@ htmlcov
 .DS_Store
 package
 dist
-*.zip
```
Comment (rymarczy) [inline .gitignore:23]: Not sure why this snuck in, but it was causing `INCOMING/MBTA_GTFS.zip` to not be added to the repo

File: src/lamp_py/ingestion/pipeline.py [hunk 3]
```diff
@@ -20,7 +20,6 @@
 
 logging.getLogger().setLevel("INFO")
 DESCRIPTION = """Entry Point For GTFS Ingestion Scripts"""
-HERE = os.path.dirname(os.path.abspath(__file__))
```
Comment (rymarczy) [inline src/lamp_py/ingestion/pipeline.py:23]: Doesn't look like this is being used anywhere.

**Review by mzappitello — APPROVED** [review]
🍰

**Pull request merged**